BookingsApi/BookingsApi.DAL/Queries/GetBookingsByCaseTypesQuery.cs

```diff
@@ -146,7 +147,7 @@ private void TryParseCursor(string cursor, out DateTime scheduledDateTime, out G
             }
         }
 
-        private IQueryable<VideoHearing> GetHearingsWithoutJudge(IQueryable<VideoHearing> hearings)
+        private static IQueryable<VideoHearing> GetHearingsWithoutJudge(IEnumerable<VideoHearing> hearings)
```

Comment:
why are we changing this to an IEnumerable?

Reply:
sonar suggestion for code smell